00_getting_started.ipynb

```diff
@@ -14,40 +14,44 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 1,
+      "source": [
+        "%%bash\n",
+        "# @title Build and install\n",
+        "export LD_LIBRARY_PATH=/usr/local/lib:${LD_LIBRARY_PATH}\n",
+        "if python -c 'import bifrost' 2>/dev/null; then\n",
+        "    echo \"Great, bifrost import seems to work.\"\n",
+        "    exit 0\n",
+        "else\n",
+        "    if python -c 'import google.colab' 2>/dev/null; then\n",
```

Comment:
Maybe a simpler way to detect that we're on colab is the environment variable `COLAB_GPU`: set to 1 if we have a GPU or 0 if not, but anyway can verify that it's set.